src/main/java/lotto/model/CalculateNumOfTickets.java

```diff
@@ -0,0 +1,37 @@
+package lotto.model;
+
+public class CalculateNumOfTickets {
+    private int numOfTickets;
+    private int purchasePrice;
+    private final int TICKET_PRICE = 1000;
+
+    public CalculateNumOfTickets() {
+    }
+
+    public CalculateNumOfTickets(int numOfTickets, int purchasePrice) {
+        this.numOfTickets = numOfTickets;
+        this.purchasePrice = purchasePrice;
+    }
+
+    public int getNumOfTickets() {
+        return numOfTickets;
+    }
+
+    public int getPurchasePrice() {
+        return purchasePrice;
+    }
+
+    //예외 - 구매금액이 양수가 아닌 경우
+    public void purchasePriceException(int purchasePrice){
+        if (purchasePrice <= 0) {
+            throw new IllegalArgumentException("구매 금액은 양수여야 합니다.");
+        }
+    }
```

Comment:
[Request Change]

메서드명은 최대한 동사로 짓는 것을 연습하면 좋을 것 같습니다.

메서드에서 어떤 것을 하는지 메서드 시그니쳐에 나타내는 것이 좋습니다.

보통 검증하는 것은 `validate{올바른 상태}` 이런식으로 짓습니다.
```suggestion
    public void validatePriceGreaterThanZero(int purchasePrice){
        if (purchasePrice <= 0) {
            throw new IllegalArgumentException("구매 금액은 양수여야 합니다.");
        }
    }
```

그리고 0또한 매직넘버로 볼 수 있습니다. 매직 넘버를 제거해주세요!

Reply:
아래와 같이 매직넘버를 상수로 변경해주었고
```java
private static final int MINIMUM_PURCHASE_PRICE=0;
```
아래와 같이 메서드명도 의도가 명확하게 바꿔주었습니다.
(조금 더 유연한 코드를 만들기 위해 ThanZero 말고 ThanMinimumPurchasePrice로 바꾸어 주었습니다.)
```java
public void validatePriceGreaterThanMinimumPurchasePrice(int purchasePrice){
        if (purchasePrice <= MINIMUM_PURCHASE_PRICE) {
            throw new IllegalArgumentException("구매 금액은 양수여야 합니다.");
        }
    }
```
###<질문>

Reply:
흠... 질문이 매우 심오하네요...
일단 저는 줄여쓰는 것을 좋아하지 않습니다.
Min어디서 갑자기 minute의 의미로 쓰일 수도 있을것같아요... 그럼 매우 헷갈리겠죠?
일단 긴 것은 의미가 잘 전달되지 않는 것에 비하면 매우 사소한 문제라고 생각합니다.

그래도 한 프로젝트 안에서는 네이밍 같은 것도 여러 방법으로 규칙을 만드는 것 같아요.
공통되는 것, 구조화되는 것, 비슷한 동작, ...
에 대해서 비슷하게 규칙적으로 네이밍을 짓는것 같아요 `validate{올바른 상태}`이것도 거기에 해당이되겠죠

Reply:
어디서 네이밍은 약어로 하지 말라는 말을 들었는 데
Min이 minute이 될 수도 있다는 게 적절한 예시인 거 같아요.

약어로 쓰면 저는 편하지만 코드를 처음 보는 사람들은 이해가 어렵겠군요.... 
그럼 메서드명은 길더라도 의미를 명확하게 지금처럼 두는 게 더 나을 거 같아요!